### DISCUSSION TITLE
Why doesn't pipe gain work for this stop?

### DISCUSSION THREAD

**Discussion by EdStauff** [GrandOrgue/grandorgue#1087] [Help] [answered]
This has me totally perplexed.  In my organ definition file I set a bunch of individual pipe gains, but they're getting ignored for one stop.  That is, when I look at the organ settings after loading, the gains I tried to set are zero (just for this one stop).  I can't see anything about this stop that would cause the gains to get ignored, when they're working for other stops in the same OD file.
The only difference I can see is that I'm trying to set negative integer gain values; all the other (working) ones are positive (unsigned) floating point values (with a decimal point).  But I tried adding a decimal point, and I tried positive values, and this one stop just doesn't want to change its pipe gains.
I've attached my OD file.
The gains that don't work are on lines 1294, 1302, and 1304.
Gains that work are on lines 826, 849, 2997, 3013, and half a dozen others.
Thanks,
-- Ed
[Veendam_Ed_10.zip](https://github.com/GrandOrgue/grandorgue/files/8299629/Veendam_Ed_10.zip)

**Comment by paolo2504** [comment]
Edward,
i had a look at your issue for curiosity as it never happened to me to deal with Gain.
I built your custom organ and converted to .orgue then i looked at stops and keys on lines you say faulty and i do see gain values seem correctly changed. From your words, i expected to see value unchanged (0).
The only difference i see in 'working' (per your words) lines (826, 849, 2997, 3013) and not working ones (1294,1302,1304) is that the Gain modification is set 'after' the Pipe lines in first case and 'before' in the second. Anycase, for me they both work as on all lines you wrote i see modifications.  Basically the lines you say not working are those with Gain after...
Paolo

![immagine](https://user-images.githubusercontent.com/50132661/158998823-7f8be56b-15bb-4948-86dc-f4fc504d930d.png)
![immagine](https://user-images.githubusercontent.com/50132661/158999446-6f9eacf9-e61f-43b1-8552-cbd941d4bf14.png)
![immagine](https://user-images.githubusercontent.com/50132661/158999714-4ca08839-2719-40bb-8038-1abb368fd5de.png)

![immagine](https://user-images.githubusercontent.com/50132661/158999034-db2a222b-9e56-4fa7-a9d0-66e68694d38f.png)
![immagine](https://user-images.githubusercontent.com/50132661/158999079-bb5265d5-5f7e-4636-93b5-12ed86b0e300.png)

**Reply by EdStauff** [reply]
Paolo, thank you very much for taking the time to look at it.
What version of G.O. are you using, and on what platform?
I'm running 0.3.1.2330 on Windows 10.
-- Ed

**Reply by paolo2504** [reply]
Hi Ed. 
I’m using latest release 3.6.4.1 on Windows 10.

**Comment by willeke1234** [comment]
Is a preset overriding the gains?

**Comment by EdStauff** [comment, marked as answer]
I installed the latest version of G.O. and the problem went away,
